lib-object-oriented/multiLine.js

```diff
@@ -1,3 +1,35 @@
+import Tag from './tag'
+import { ASTERISK, HYPHEN, NUMB } from './token'
+
 export default class MultiLine {
-  //Take a bunch of lines and output a single string of html
+  constructor( lineCollection, type ) {
+    this.lineCollection = lineCollection
+    this.listType = type
+    this.htmlOutput = ''
+    this.tagType = null
+  }
+
+  determineTagType = () => {
```

Comment:
this function is updating the property `tagType` of the instance. Can the function name reflect the behavior?